### DISCUSSION TITLE
Welcome to Gravity Discussions!

### DISCUSSION THREAD

**Discussion by marcobambini** [marcobambini/gravity#338] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KingKey-X** [comment]
please to support c++ embedded

**Comment by marcobambini** [comment]
@KingKey-X please take a look at this example https://github.com/marcobambini/gravity/tree/master/examples/GravityCpp

**Reply by KingKey-X** [reply]
Thanks
